src/inference/tests/functional/blob_copy_test.cpp

```diff
@@ -80,7 +80,7 @@ InferenceEngine::Blob::Ptr createBlob(InferenceEngine::Precision precision,
 size_t GenerateRandom(size_t elem) {
     size_t result;
     do {
-        result = std::floor(std::rand() / static_cast<float>(RAND_MAX * elem));
+        result = (size_t)std::floor(std::rand() / static_cast<float>(RAND_MAX * elem));
```

Comment:
Please use static cast instead of C style cast

Reply:
done